.github/workflows/configurable-benchmark-test-self-hosted.yaml

```diff
@@ -83,14 +83,15 @@ jobs:
           sudo apt-cache policy docker-ce
           sudo apt install -y docker-ce
           sudo systemctl status docker
+          sudo mkdir -p ~/.kube
+          sudo chmod 777 ~/.kube
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+          sudo install minikube-linux-amd64 /usr/local/bin/minikube
 
-      - name: Setup Kubernetes
-        uses: manusa/actions-setup-minikube@v2.7.1
-        with:
-          minikube version: 'v1.23.2'
-          kubernetes version: 'v1.23.2'
-          driver: docker
-          start args: --disk-size 10000mb
```

Comment:
Any reason for replacing this action for a curl download then a manual install?

Reply:
Self-hosted runners didn't allow the action to setup k8s, so curl method works for now